src/library.js

```diff
@@ -2296,6 +2296,16 @@ addToLibrary({
 
   $noExitRuntime: "{{{ makeModuleReceiveExpr('noExitRuntime', !EXIT_RUNTIME) }}}",
 
+  // functions called during startup
```

Comment:
Perhaps this comment could be a little more descriptive?   I'm not sure if this is correct but maybe "Function the are called once the runtime is initialized and static constructors have been called".